### DISCUSSION TITLE
Problem on upload proof of your academic status

### DISCUSSION THREAD

**Discussion by benjamimWalker** [community/community#62347] [GitHub Education]
I am facing some issues taking the photo from my smartphone, I can't change to back camera, it always stays on front camera, it is hard for me to take the picture, all of them are horribles because of that.

**Comment by dakshsinghrathore** [comment]
1. _Scan Your Document_ : If you have access to a scanner, you can scan your academic status document and save it as an image file (e.g., JPG or PNG). This will give you a high-quality image that you can then upload to GitHub Education
2. _Contact GitHub Support_ : If none of the above options work for you, consider reaching out to GitHub Support. Explain your situation and the difficulties you're facing with taking a photo. They might be able to provide additional guidance or assistance.

**Comment by lydhr** [comment]
No one answers my GitHub Support ticket either. Using back camera or uploading attachment is still not working at all. I cannot believe this is a Github website.

**Reply by lydhr** [reply]
@dakshsinghrathore

**Reply by dakshsinghrathore** [reply]
mail them regarding the issue, and be patient.

**Reply by lydhr** [reply]
It has been 3 weeks since I submitted the ticket. Sorry, I thought you were the staff.
